torch/utils/backcompat/broadcast/warning/__init__.py

```diff
@@ -0,0 +1,18 @@
+import sys
+from torch._C import _set_backcompat_broadcast_warn
+from torch._C import _get_backcompat_broadcast_warn
+
+
+class Warning:
+    def __init__(self):
+        pass
+
+    def set_enabled(self, value):
+        _set_backcompat_broadcast_warn(value)
+
+    def get_enabled(self):
+        return _get_backcompat_broadcast_warn()
+
+    enabled = property(get_enabled, set_enabled)
+
+sys.modules[__name__] = Warning()
```

Comment:
1. Don't use old style classes (inherit from `object` explicitly)
2. Wouldn't it be simpler to have a `warning` variable in `torch.utils.backcompat.broadcast` instead of doing this hack?